src/NmeaParser.Shared/Nmea/Interfaces/ILatLng.cs

```diff
@@ -0,0 +1,34 @@
+﻿//
+// Copyright (c) 2016 Morten Nielsen
+//
+// Contributors:
+// Stephen Kennedy, Copyright (c) 2016 Gloucester Software Ltd.
+//
+// Licensed under the Microsoft Public License (Ms-PL) (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://opensource.org/licenses/Ms-PL.html
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+namespace NmeaParser.Nmea
+{
+    public interface ILatLng
```

Comment:
I don't really get what purpose this serves. Location on each message can mind wildly different things, so there's not much benefit in a common interface. You have to understand the message to understand the value.
Also naming should be more explicit (ILatitudeLongitude - but that's still a poor name - ILocation but then that's arbitrary)